.tekton/push.yaml

```diff
@@ -21,7 +21,8 @@ spec:
       value: Dockerfile
     - name: infra-deployment-update-script
       value: |
-        sed -i -e 's|\(https://github.com/redhat-appstudio/release-service/config/default?ref=\)\(.*\)|\1{{ revision }}|' -e 's/\(newTag: \).*/\1{{ revision }}/' components/release/kustomization.yaml
+        sed -i -e 's|\(https://github.com/redhat-appstudio/release-service/config/default?ref=\)\(.*\)|\1{{ revision }}|' -e 's/\(newTag: \).*/\1{{ revision }}/' components/hacbs/release-service/base/kustomization.yaml
```

Comment:
config/kcp not config/default and also when you apply this script on https://github.com/redhat-appstudio/infra-deployments/pull/815 then you are missing `"` at the end

Reply:
Nice catch. Not following the missing `"` though

Reply:
Removing the `"`s in https://github.com/redhat-appstudio/infra-deployments/pull/815

Reply:
`- "https://github.com/redhat-appstudio/release-service/config/kcp?ref=69f4ae703bb51b356b9973133596bba235147245"`

is converted to 
`- "https://github.com/redhat-appstudio/release-service/config/kcp?ref={{ revision }}`

Missing `"` in the end of line, other services do not put quotes to the url